testplan/web_ui/testing/src/AssertionPane/AssertionTypes/BasicAssertion.js

```diff
@@ -27,60 +27,59 @@ import { prepareBasicContent } from "./basicAssertionUtils";
  * the data to be displayed. It returns an object that fills the aforementioned
  * grid with data.
  */
-class BasicAssertion extends Component {
-  render() {
-    const {
-      preTitle,
-      preContent,
-      leftTitle,
-      rightTitle,
-      leftContent,
-      rightContent,
-      postTitle,
-      postContent,
-    } = prepareBasicContent(this.props.assertion);
+function BasicAssertion({ assertion }) {
+  const {
+    preTitle,
+    preContent,
+    leftTitle,
+    rightTitle,
+    leftContent,
+    rightContent,
+    postTitle,
+    postContent,
+  } = prepareBasicContent(assertion);
 
-    return (
-      <Fragment>
-        <Row>
-          <Col lg="12">
-            <strong>{preTitle}</strong>
-          </Col>
-        </Row>
-        <Row>
-          <Col lg="12" className={css(styles.contentSpan)}>
-            {preContent}
-          </Col>
-        </Row>
-        <Row>
-          <Col lg="6">
-            <strong>{leftTitle}</strong>
-          </Col>
-          <Col lg="6">
-            <strong>{rightTitle}</strong>
-          </Col>
-        </Row>
-        <Row>
-          <Col lg="6" className={css(styles.contentSpan)}>
-            <span>{leftContent}</span>
-          </Col>
-          <Col lg="6" className={css(styles.contentSpan)}>
-            <span>{rightContent}</span>
-          </Col>
-        </Row>
-        <Row>
-          <Col lg="12">
-            <strong>{postTitle}</strong>
-          </Col>
-        </Row>
-        <Row>
-          <Col lg="12" className={css(styles.contentSpan)}>
-            {postContent}
-          </Col>
-        </Row>
-      </Fragment>
-    );
-  }
+  console.log("BasicAssertion has been rendered.");
```

Comment:
Intentional or just local use and left in?

Reply:
Just for testing purposes, will be removed in final version.